context/app/static/js/components/types.ts

```diff
@@ -38,7 +38,9 @@ export interface Entity {
   contacts: ContactAPIResponse[];
   contributors: ContributorAPIResponse[];
   created_timestamp: number;
+  /** @deprecated Use `ancestor_ids` and `useEntitiesData` instead */
   ancestors: Entity[];
+  ancestor_ids: string[];
   // eslint-disable-next-line no-use-before-define -- Donor is defined later in the file and extends Entity
   donor: Donor;
   descendant_counts: { entity_type: Record<string, number> };
```

Comment:
Should we add like comments for `descendants` and `descendant_counts` or do we plan to keep those for now? I don't think we have the immediate variants in the types.

Reply:
I think we do plan to keep `descendant_counts`,for the time being, but good call on adding this comment for `descendants`!